python/lsst/daf/butler/core/datasets.py

```diff
@@ -148,6 +148,18 @@ def componentTypeName(self, component):
             return self.nameWithComponent(self.name, component)
         raise KeyError("Requested component ({}) not understood by this DatasetType".format(component))
 
+    def lookupNames(self):
+        """Names to use when looking up this datasetType in a configuration.
+
+        The names are returned in order of priority.
+
+        Returns
+        -------
+        names : `tuple` of `str`
+            Tuple of the `DatasetType` name and the `StorageClass` name.
+        """
+        return (self.name, *self.storageClass.lookupNames())
```

Comment:
We have multiple lookup names for storageclasses now? Is this to deal with `ImageI`, `ImageF`, etc?

Reply:
No, but to retain compatibility with the `lookupNames()` interface it returns a tuple with a single element in it that needs to be unpacked.